### PR TITLE
[TechDocs] Enhance PlantUML support in the techdocs container to include external puml or pu files

### PR DESCRIPTION
**Pull request by lavanya-sainik-ericsson**
I have created a pull request for the issue I have outlined on backstage. Please refer here : https://github.com/backstage/backstage/issues/20184.

[awanlin](https://github.com/awanlin) As discussed in the issue thread, looping you here as well to have a look.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,6 +26,11 @@ RUN pip install --upgrade pip && pip install mkdocs-techdocs-core==1.2.2
 #   error (OSError: [Errno 8] Exec format error: 'plantuml') by using the
 #   following RUN command instead:
 #   RUN echo '#!/bin/sh\n\njava -jar '/opt/plantuml.jar' ${@}' >> /usr/local/bin/plantuml
+
+#   When adding TechDocs with plantUml diagrams, to refer external puml or pu files in any markdown file,
```
Comment (awanlin) [inline Dockerfile:30]:
```suggestion
#   When adding TechDocs with PlantUML diagrams, to refer external puml or pu files in any markdown file,
```

File: Dockerfile [hunk 2]
```diff
@@ -26,6 +26,11 @@ RUN pip install --upgrade pip && pip install mkdocs-techdocs-core==1.2.2
...
+#   eg. '!include <referencedFileName.puml>', need to include diagrams directory eg. docs in the classpath.
```
Comment (awanlin) [inline Dockerfile:31]:
```suggestion
#   eg. '!include <referencedFileName.puml>', you'll need to include the diagrams directory eg. docs in the classpath.
```

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks again for this contribution @lavanya-sainik-ericsson, just left a couple of quick changes 👍

**Comment by lavanya-sainik-ericsson** [comment]
Hello [awanlin](https://github.com/awanlin), 

Thanks for your comments. Have done those changes. Let me know if any more corrections is needed.

**Review by awanlin — APPROVED** [review]
Looks good to me @lavanya-sainik-ericsson, you'll need someone from @backstage/techdocs-maintainers to review and approve, they will also do the merge

**Comment by lavanya-sainik-ericsson** [comment]
Hey [awanlin](https://github.com/awanlin), 

Thanks a lot for your feedback and approval. Sure can you share how to request reviews from @backstage/techdocs-maintainers here & request their attention & valuable feedback ?

**Review by johnphilip283 — APPROVED** [review]
Thank you for the change - approved!

**Pull request merged**